sys/sys/elf_common.h

```diff
@@ -1103,6 +1109,9 @@ typedef struct {
 #define	R_MORELLO_TLS_TPREL128	59398
 #define	R_MORELLO_FUNC_RELATIVE	59400
 #define	R_AARCH64_FUNC_RELATIVE	59401
+#define	R_MORELLO_TLS_TGOT_SLOT	59916
```

Comment:
These are currently in the "reserved for private Morello experiments" range; we should reserve real relocations at some point before merging